examples/battery_pool.py

```diff
@@ -13,8 +13,7 @@
 from frequenz.sdk import microgrid
 from frequenz.sdk.actor import ResamplerConfig
 
-HOST = "microgrid.sandbox.api.frequenz.io"  # it should be the host name.
-PORT = 62060
+MICROGRID_API_URL = f"grpc://microgrid.sandbox.api.frequenz.io:62060"
```

Comment:
this might not need to be a f-string
```suggestion
MICROGRID_API_URL = "grpc://microgrid.sandbox.api.frequenz.io:62060"
```

Reply:
Thanks, fixed!